R/add_base_section.R

```diff
@@ -1,12 +1,12 @@
 #' Add selected sections to outline
 #'
-#' @param x inherited 'custom_sections' parameter from create_template
+#' @inheritParams create_template 'custom_sections' parameter is from create_template.R
```

Comment:
You don't need to add a description following the @inheritParams create_template. The call essentially replicates the description for any of the common parameters from.

Here is where the function gets used in create_template and it's just a way to add either all of the default sections or the ones defining custom_sections

https://github.com/nmfs-ost/asar/blob/dd56fd7c857cee1c9b9c95ed390ac5fd2785ca12/R/create_template.R#L744-L751